.github/workflows/ci-weekly.yml

```diff
@@ -136,7 +139,7 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-
+        
```

Comment:
```suggestion
```

Reply:
@chrisdholt this is by design.  There should be an empty row between steps/tasks for easier readability.